.github/workflows/build-main.yml

```diff
@@ -90,6 +92,7 @@ jobs:
         python-version: 3.9
     - name: Install Dependencies
       run: |
+        git tag
```

Comment:
Question: will this just show the single tag that is being deployed or the whole taglist?